packages/web/src/lib/wallectconnect-mode.ts

```diff
@@ -157,6 +157,21 @@ export class WallectConnectSession {
 		}
 	}
 
+	/**
+	 * @description Signs a Logic Signature transaction
+	 * @param transaction algosdk.Transaction object
+	 * @param logicSig Logic Sig Account
+	 * @returns Returns txID and blob object
+	 */
+	signLogicSignatureTxn(transaction: Transaction, logicSig: algosdk.LogicSigAccount): { txID: string, blob: Uint8Array } {
```

Comment:
let's apply similar changes as above (txn -> tx, Signature -> Sig, update description). Same for other functions below.

Reply:
okay